scripts/submitHistoFiller_parallel.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import argparse
+import datetime
+
+parser = argparse.ArgumentParser(description='Command line parser of plotting options')
+parser.add_argument('--cfg'     , dest='cfg'     ,            default=None, help='name of the main cfg to run')
+parser.add_argument('--tag'     , dest='tag'     ,            default=None, help='name of working space (defaults to timestamp)')
+parser.add_argument('--njobs'   , dest='njobs'   , type=int , default=10  , help='number of jobs for parallelization')
+parser.add_argument('--isCondor', dest='isCondor', type=bool, default=True, help='run on condor or not')
+args = parser.parse_args()
+
+
+if not args.cfg:
+    print "** Please provide a cfg name: --cfg <relative path from KLUBanalysis/ or absolute path>"
+    sys.exit()
+
+outDir = datetime.datetime.now().strftime('%Y.%m.%d_%H.%M.%S')
+if args.tag:
+    outDir = args.tag
+
+here = os.getcwd()
+
+program = 'testAnalysisHelper.exe'
+
+proto = 'filler_' ## job .sh fie name
+logproto = 'log_' ## job .sh fie name
+
```

Comment:
```suggestion
os.system('mkdir ' + outDir)
```

I think there should be an `mkdir` here.

Reply:
no, the `mkdir` is now in `launchHistoFiller.py` where we can set the proper name (including tag/year/channel)

Reply:
I did not check there, perfect then